pyopencl/_monkeypatch.py

```diff
@@ -259,7 +259,7 @@ def kernel_set_arg_types(self: _cl.Kernel, arg_types):
                     devs=self.context.devices))
 
 
-def kernel_get_work_group_info(self: _cl.Kernel, param: int, device: _cl.Device):
+def kernel_get_work_group_info(self: _cl.Kernel, param: int, device: _cl.Device) -> int:
```

Comment:
This isn't correct. For example `CL_KERNEL_COMPILE_WORK_GROUP_SIZE` returns a sequence of integers:

https://github.com/inducer/pyopencl/blob/b3bb32d6527bb7bae407004da9e7903066d3405e/src/wrap_cl.hpp#L5088-L5095

`object` is probably the right thing here.

Reply:
Yeah, thought that would have been too easy :( Changed it to `object` and added some more checks and casts in https://github.com/inducer/pyopencl/compare/3d7d45e82efd6e92488dad748cf9781e27b32a00..bfc1af2939357dd2ef6ae4dbe45bfd4869f29412.